src/Schema/Events/FilterAndSort.ts

```diff
@@ -85,3 +85,25 @@ export interface PriceDatabaseFilterParamsChanged {
   current: string
   changed: string
 }
+
+/**
+ * A user selects a recent price range from the price filter screen
+ *
+ * This schema describes events sent to Segment from [[selectedRecentPriceRange]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "selectedRecentPriceRange",
+ *    context_module: "recentPriceRanges",
+ *    context_screen_owner_type: "artworkPriceFilter",
+ *    collector_profile_sourced: true
+ *  }
+ * ```
+ */
+export interface SelectedRecentPriceRange {
+  action: ActionType.selectedRecentPriceRange
+  context_module: ContextModule.recentPriceRanges
+  context_screen_owner_type: OwnerType.artworkPriceFilter
+  collector_profile_sourced: boolean 
+}
```

Comment:
**question**: @bbaiyr should we also send the specified price range? will this information be useful?

Reply:
I think the applied price range should show up in the existing `CommercialFilterParamsChanged` event so not sure if we need it here, especially because the user hasn't actually applied the filter yet.

Reply:
Yes, but we tracks this event only if the user clicked the "Show Results" button